llama_stack/providers/adapters/inference/tgi/config.py

```diff
@@ -12,9 +12,13 @@
 
 @json_schema_type
 class TGIImplConfig(BaseModel):
-    url: str = Field(
-        description="The URL for the TGI endpoint (e.g. 'http://localhost:8080')",
-    )
+    host: str = "localhost"
```

Comment:
I think we should add protocol here also (and in remote provider also) with default = http and option for https :)

Reply:
done!